tools/llm_bench/task/image_generation.py

```diff
@@ -95,13 +95,85 @@ def run_image_generation(image_param, num, image_id, pipe, args, iter_data_list,
     stable_diffusion_hook.clear_statistics()
 
 
+def run_image_generation_genai(image_param, num, image_id, pipe, args, iter_data_list, proc_id, mem_consumption, tokenizer):
+    set_seed(args['seed'])
+    input_text = image_param['prompt']
+    image_width = image_param.get('width', DEFAULT_IMAGE_WIDTH)
+    image_height = image_param.get('height', DEFAULT_IMAGE_HEIGHT)
+    nsteps = image_param.get('steps', DEFAULT_INFERENCE_STEPS if 'lcm' not in args["model_name"] else LCM_DEFAULT_INFERENCE_STEPS)
```

Comment:
probably this part can be generalized in future (there are except lcm, there are also turbo models that may work with 1 step and hyper 2-8)

Reply:
why not to rely on default `inference_steps` defined in pipeline itself?

Reply:
it is not always correct, e.g. adding lcm lora to pipeline keep pipeline default steps like in original while it should be manually reduced